File: Max-min arrays.py
def solve(arr):
    sort_arr = sorted(arr)
    result = [[sort_arr[len(sort_arr)-1-i],sort_arr[i]] for i in range(len(sort_arr)//2)]
    if len(sort_arr)%2!=0:
        result.append([sort_arr[len(sort_arr)//2]])
    return sum(result,[])
    
        
        

File: Max-min arrays/Max-min arrays.js
function solve(arr){
    const rev_arr = [...arr].sort((a,b) => a-b)
    let result = [];
    for(let i=0;i<Math.floor(arr.length/2);i++){
      result.push(rev_arr[arr.length-1-i])
      result.push(rev_arr[i])
    }
    if(arr.length%2!=0)result.push(rev_arr[Math.floor(arr.length/2)])
    return result
};
